Gemfile.lock

```diff
@@ -1,82 +0,0 @@
-PATH
```

Comment:
Do you want to include a new Gemfile.lock?

Reply:
Sure. Can you help to generate the lock file ?

Reply:
@ajfarkas  Added Gemfile.lock  
Gitlab Ci 

https://gitlab.login.gov/lg/identity-validations/-/pipelines/115391